mkdocs.yml

```diff
@@ -30,6 +30,9 @@ pages:
   - 'How to use PL to learn student names': 'howToLearnNames.md'
   - 'FAQ': 'faq.md'
   - 'API': 'api.md'
+- Getting Started:
+  - 'Requesting your course space': 'requestCourse.md'
+  - 'Creating content in the browser': 'getStarted.md'
```

Comment:
Can we move this up to the top?

Reply:
I am so puzzled... I made these changes yesterday, I am not sure how it did not get added. I changed it again now.